Automattic/sensei
# Fix empty jQuery datepicker UI box showing in the footer

### Changes proposed in this Pull Request

This PR updates our jQuery UI CSS which fixes the empty datepicker UI box that is displayed on pages that have a datepicker.

### Testing instructions

* Go to Sensei LMS -> Reports -> Students
* Make sure the empty datepicker box is not displayed.
* Make sure the datepicker inputs are working.

### Screenshot

![CleanShot 2022-03-21 at 15 24 26@2x](https://user-images.githubusercontent.com/329356/159260761-8169df9c-aa03-4394-9b24-7e0ab1278467.png)

### Context

Spotted in #4932
